server/src/index.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express');
+
+const port = process.env.PORT || 5000;
+
+const app = express();
+
```

Comment:
Can we create at least 1 route for '/' with response 'hello world' or 'coming soon'.

Reply:
This is also done.